Description: make the error message more helpful on Ubuntu
Forwarded: no
Updated: 2014-02-17

Index: virt-manager-1.4.1/virtManager/engine.py
===================================================================
--- virt-manager-1.4.1.orig/virtManager/engine.py	2017-03-09 18:11:10.653005288 +0100
+++ virt-manager-1.4.1/virtManager/engine.py	2017-03-09 18:11:10.653005288 +0100
@@ -671,7 +671,10 @@
                           "regular user. Try running as root.")
                 show_errmsg = False
             elif re.search(r"libvirt-sock", tb):
-                hint += _("Verify that the 'libvirtd' daemon is running.")
+                hint += _("Verify that:\n" +
+                         " - The 'libvirt-bin' package is installed\n" +
+                         " - The 'libvirtd' daemon has been started" +
+                         (conn.is_qemu_system() and "\n - You are member of the 'libvirtd' group" or ''))
                 show_errmsg = False
 
         probe_connection = self.conns[conn.get_uri()]["probeConnection"]
